script-library/docker-debian.sh

```diff
@@ -15,8 +15,11 @@ TARGET_SOCKET=${3:-"/var/run/docker.sock"}
 USERNAME=${4:-"automatic"}
 USE_MOBY=${5:-"true"}
 DOCKER_VERSION=${6:-"latest"}
+DOCKER_COMPOSE_PLUGIN_VERSION=${7:-"latest"}
```

Comment:
Do we want to update the features.json and .env file with these new flags?

Reply:
Yeah I was debating that.  Some of these seem like they'd be "hidden" fields as we have UX that allows you to select options. Probably does make sense to add them at the end of the day.